packages/devtools/lib/src/model/model.dart

```diff
@@ -79,9 +79,7 @@ class App {
     js.context['devtools'] = binding;
   }
 
-  Future<void> devToolsReady(dynamic message) async {
-    _sendNotification('app.devToolsReady', message);
```

Comment:
hmm. I'm not sure this is the right place to delete it.
I think test code is depending on the 'app.devToolsReady' event.
Can we just tweak things so it doesn't get displayed in the console?

Reply:
Yes, just found that out - travis failed.  I'm not sure _sendNotification should ever print to the console?  Think this print should be commented out for now.  With a TODO number of other things also print to console in model.dart.